docs/overview.md

```diff
@@ -0,0 +1,59 @@
+# What is the MyST stack?
+
+This section provides a high-level overview of the main concepts and tools in the MyST ecosystem. See the [guiding principles of the MyST ecosystem](../guiding-principles.md) for some more high-level context.
+
+## A brief overview of the MyST Stack
+
+Here is a brief description of the workflow that is involved in the MyST ecosystem and the technical pieces that are involved at each step. You can hover over (or click) the links in the list below to see more information about each one.
+
+- An author writes content in {term}`MyST Markdown` in a file.
+- A {term}`MyST Document Engine` parses one or more files and builds them into structured data called the {term}`MyST AST`.
+- The {term}`MyST AST` conforms to the {term}`MyST Specification`, which provides a lot more information and metadata.
+- A {term}`MyST Renderer` can then transform the {term}`MyST AST` into different kinds of outputs (most commonly HTML, or PDF via Latex/Typst). It uses a {term}`MyST Theme` to do so.
+- The [official MyST CLI](https://mystmd.org) contains both a parser and a renderer so that you can go from a local file to an output with one application. 
+
+Here's a diagram showing how these all relate to one another:
+
+:::{figure} images/myst-diagram.svg
+An overview of some major parts of the MyST stack and how they relate to one another.
+In the most common workflow, a text file written in MyST Markdown is parsed by the {term}`MyST Document Engine` which outputs {term}`MyST AST` that follows the {term}`MyST Specification` (usually a JSON file). This MyST AST can be rendered into many different kinds of outputs, most-commonly HTML or PDF. This usually done by the MyST Document Engine, but can be done by any application that understands how to parse and use the MyST Specification.
```

Comment:
The parenthetical "usually a JSON file" should probably glo after "MyST AST" and not after "MyST specification".